src/Helpers/JWKFetcher.php

```diff
@@ -98,7 +98,7 @@ public function getKeys($jwks_url)
     /**
      * Fetch x509 cert for RS256 token decoding.
      *
-     * TODO: Deprecate, use $this->getFormatted()
+     * @deprecated 5.6.0, use $this->getKeys().
```

Comment:
Intentionally changing replacement from getFormatted to getKeys?  Or copy-paste from line 154?

Reply:
@damieng - Intentional. Thank you for checking :)